FreeRTOS/FreeRTOS
# Update single core CMBC and unit test

Description
-----------
This PR fixes CBMC and unit test for single core FreeRTOS in the [FreeRTOS-Kernel PR](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/716).

* xYieldPendings and xIdleTaskHandles are now an array. Update in FreeRTOS unit test.
* Update CBMC patches.

Test Steps
-----------
Before this patch there will be compilation error when building unit test for FreeRTOS and CBMC.


Checklist:
----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
